server/etcdmain/etcd.go

```diff
@@ -272,6 +272,6 @@ func checkSupportArch() {
 		return
 	}
 
-	lg.Error("running etcd on unsupported architecture since ETCD_UNSUPPORTED_ARCH is set", zap.String("arch", runtime.GOARCH))
+	lg.Error("Refusing running etcd on unsupported architecture since ETCD_UNSUPPORTED_ARCH is not set", zap.String("arch", runtime.GOARCH))
```

Comment:
"Refusing to run etcd... ", sounds a little more English to me. Please also sign your commit with "git commit -s".